modules/core/src/main/java/org/apache/ignite/internal/commandline/CommonArgParser.java

```diff
@@ -336,4 +374,52 @@ private String securityWarningMessage(String password) {
 
         return String.format(pwdArgWarnFmt, password, password);
     }
+
+    /**
+     * Extracts user attributes from attribute string.
+     *
+     * @param attrMap {@code Map} Attribute map.
+     * @param attrs {@code String} Attribute string.
+     */
+    private void extractAttributesFromString(Map<String, String> attrMap, String attrs) {
+        final int partsOfAttrStr = 2;
+
+        for (String attr : attrs.split(",")) {
+            if (!attr.contains("=")) {
+                logger.warning(String.format("Failed to parse attribute %s", attr));
+
+                continue;
+            }
+
+            String[] keyVal = attr.split("=", partsOfAttrStr);
+
+            if (!attrMap.containsKey(keyVal[0]))
+                attrMap.put(keyVal[0], keyVal[1]);
+        }
+    }
+
+    /**
+     * Extracts user attributes from a given file.
+     *
+     * @param attrMap {@code Map} Attribute map.
+     * @param path {@code String} Path to the file.
+     */
+    private void extractAttributesFromFile(Map<String, String> attrMap, String path) {
+        Properties attrs = new Properties();
+
+        try (InputStream is = new FileInputStream(new File(path))) {
+            attrs.load(is);
+        } catch (Exception e) {
+            logger.log(Level.SEVERE, "Failed to read property file: " + path, e);
+
+            throw new RuntimeException(e);
+        }
+
+        attrs.forEach((key, value) ->
+                {
```

Comment:
K&R bracing and indentation style should be used. { starts on the same line as the opening block statement.
See https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-BracesandIdentation

Reply:
fixed